### DISCUSSION TITLE
NvimTree actions popup so noise after updating to Lazyvim 14.1.0

### DISCUSSION THREAD

**Discussion by vqcuong** [LazyVim/LazyVim#5057] [General]
I've updated to the latest version of Lazyvim (v14.1.0). Previously, I use NvimTree as the default file explorer and when acting on NvimTree like delete or rename files/folders just use dressing popup and without any suggestion. However, after updating to latest version, the action popup also change to noice UI and there are many suggestion showing whenever typing. How to make it back to the last version?

**Comment by folke** [comment]
```lua
{ "snacks.nvim",  opts = { input = { enabled = false } } }
```
